addons/account_edi/models/account_move.py

```diff
@@ -193,7 +193,7 @@ def default_grouping_key_generator(tax_values):
                     'tax_id': self.env['account.tax'].browse(tax_res['id']),
                     'tax_repartition_line_id': self.env['account.tax.repartition.line'].browse(tax_res['tax_repartition_line_id']),
                     'base_amount': sign * invoice_line.company_currency_id.round(tax_res['base'] * rate),
-                    'tax_amount': sign * invoice_line.company_currency_id.round(tax_res['amount'] * rate),
```

Comment:
I think if it is round by line, it is better to still send it with the rounding (and in fact then the same should apply for the currency values)  Just to not break the people still in round by line.  @jbw-odoo

Reply:
ok so what i did before ...

Reply:
Sorry for that.  Initially, I meant just removing the round when round globally instead of adding the precision.  @jbw-odoo

Reply:
@jbw-odoo liko so ?

Reply:
@jco-odoo like this ?

Reply:
@smetl This is done in account_edi in v14 as we do not have the 2 compute methods here.  It seems that compute_all checks the rounding method as well, so with this simple check we actually support both rounding methods easily.  Just wanted to check with you as well.

Reply:
I'm not sure you need to round this amount then... This is already managed by the compute_all method so `'tax_amount': sign * tax_res['amount'] * rate,` is enough I think

Reply:
`rate` is of course not rounded, but I suppose you could say it is still rounded by line and the round will be done anyways when saving?  @smetl